docs/aws/guides/lookup-tables.md

```diff
@@ -0,0 +1,126 @@
+# Lookup Tables with AWS
+
+If you use psoxy to send pseudonymized data to Worklytics and later wish to re-identify the data
+that you export from Worklytics to your premises, you'll need a lookup table in your data warehouse
+to JOIN with that data.
+
+![AWS data flow](lookup-table-data-flow.png)
```

Comment:
picky detail about image contents: it has too much whitespace below the actual diagram... it's going to create a bad looking result